spec/models/group_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+RSpec.describe Group, model: :Group do
+  describe 'Tests for Group model validation' do
+    @user = User.create(name: 'Test', email: 'termus96@gmail.com', password: '123456')
+
+    subject { Group.new(author: @user, name: 'Test operation') }
+    before { subject.save }
+
+    it 'name should be present and not blank' do
+      subject.name = nil
+      expect(subject).to_not be_valid
+      subject.name = ''
+      expect(subject).to_not be_valid
+    end
+
+    it 'name should not exceed 50 characters' do
+      subject.name = 'test' * 50
+      expect(subject).to_not be_valid
+    end
+  end
```

Comment:
- [ ] Nice adding tests but you should add integration tests to cover your app as required 
- Screenshot of the requirements 
![tests](https://user-images.githubusercontent.com/61790587/195636177-f8ece6a6-7622-4830-a4f0-0c725b80bb22.png)